tailcfg/tailcfg.go

```diff
@@ -2214,6 +2214,9 @@ const (
 
 	// NodeAttrsTailFSAccess enables accessing shares via TailFS.
 	NodeAttrsTailFSAccess NodeCapability = "tailfs:access"
+
+	// NodeAttrsSuggestExitNode labels an exit node suggestion candidate.
+	NodeAttrsSuggestExitNode NodeCapability = "suggest-exit-node"
```

Comment:
So at most one node has this label?

Reply:
No - we will tag each potential exit node candidate this node attribute as a way for the client to know of control's recommendations

Reply:
Let's reword this then. Maybe something like: ?

```
// NodeAttrsSuggestExitNode is applied to each exit node which the control plane has determined
// is a recommended exit node.
```